### DISCUSSION TITLE
"Error while loading shared libraries" after trying to i-use-arch-btw

### DISCUSSION THREAD

**Discussion by consler** [overmighty/i-use-arch-btw#19] [Q&A] [answered]
After I run `i-use-arch-btw -h` I get
`i-use-arch-btw: error while loading shared libraries: libiuab.so.0.1: cannot open shared object file: No such file or directory
`
After I did some research I ran `sudo find / -name libiuab.so.0.1` which gave

`find: ‘/run/user/1000/doc’: Permission denied`
`/home/consler/i-use-arch-btw/build/lib/libiuab.so.0.1 `
`/usr/local/lib/libiuab.so.0.1`

Which didn`t really give ME any information so I'm asking for help here.

Also a few days ago I reinstalled the OS(arch linux btw)

**Comment by overmighty** [comment, marked as answer]
The [dynamic linker/loader](https://man.archlinux.org/man/ld.so.8) is probably not searching for libraries in /usr/local/lib. Try this:

```bash
export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
i-use-arch-btw -h
```

You can also use the [i-use-arch-btw-git AUR package](https://aur.archlinux.org/packages/i-use-arch-btw-git) which installs into /usr like other packages, so it doesn't require you to change the `LD_LIBRARY_PATH` environment variable. When you install the project manually with CMake commands, it installs into /usr/local by default.
